### ISSUE TITLE
Wrong OAuth2 related configuration

### ISSUE THREAD

**Issue by siddharth3322** [unity-plugins/google-services-unity#18]
Basically I was facing issue at the time of Sign up process.
Here are two images to give you more idea about my problem:

Client id error
![clientiderror](https://user-images.githubusercontent.com/3082151/29740295-d3396e28-8a70-11e7-8b14-46fed24e58e4.png)

Wrong OAuth2
![wrongoaouth2](https://user-images.githubusercontent.com/3082151/29740297-db642e12-8a70-11e7-909b-2ef648e4792a.png)

For this reason, I was getting failed during sign up time. If I upload new signed APK file within same game app then its get uploaded successfully so where exactly problem running that I can't able to understand.

Any kind of more information, I am ready to provide so you get more information about problem, I was facing. Please give me some help into this.

**Comment by unity-plugins** [comment]
There may same error with your gps config and your sign cert.

**Comment by siddharth3322** [comment]
Basically its my simple candy crush kind of game and I want to implement leaderboard functionality within that.

Basically what is going wrong that I can't able to understand!!
which ever screenshots you want I am ready to provide but please give me some hint into this.

**Comment by unity-plugins** [comment]
https://github.com/unity-plugins/google-services-unity/blob/master/Assets/Plugins/Android/google-services.json
https://github.com/unity-plugins/google-services-unity/blob/master/Assets/Plugins/Android/AndroidManifest.xml
https://github.com/unity-plugins/google-services-unity/blob/master/Assets/Plugins/Android/res/values/ids.xml

get google-services.json from firebase,and edit ids.xml and androidmanifest.xml

**Comment by siddharth3322** [comment]
Basically I am not using any services of Firebase, I am just trying to use Google Play Service leaderboard. And for that I am trying to do sign up in play services.

Anything related to Firebase, I have not touched.

**Comment by unity-plugins** [comment]
Yes,You need get google-services.json and ids.xml information too
